EXTRACTOR_SERVICE/src/model/tesseract/extractor.ts

```diff
@@ -16,10 +19,35 @@ export abstract class Extractor {
 	 * @protected method
```

Comment:
Maybe you can specify what this @protected method does, or just delete this comment

Reply:
It has a comment. In line 18/17